mods/ra/maps/allies-09a/allies09a.lua

```diff
@@ -0,0 +1,128 @@
+	end)
+end
+
+
+RescueFailed = function()
+	Media.PlaySpeechNotification(player, "ObjectiveNotMet")
+	player.MarkFailedObjective(KosyginSurviveObjective)
+end
+
+CreateKosygin = function()
+	player.MarkCompletedObjective(UseSpyObjective)
+	Media.PlaySpeechNotification(player, "ObjectiveMet")
+	kosygin = Actor.Create(kosyginType, true, { Location = KosyginSpawnPoint.Location, Owner = player })
+	Trigger.OnKilled(kosygin, RescueFailed)
+	ExtractObjective = player.AddPrimaryObjective("Extract Kosygin.")
+	Trigger.AfterDelay(DateTime.Seconds(1), function() Media.PlaySpeechNotification(player, "TargetFreed") end)
+end
+
+
+InfiltrateForwardCenter = function()
+	Trigger.OnInfiltrated(USSRFC, function()
+		if contectKosygin == false then
+			contectKosygin = true
+			CreateKosygin()
+		end
+	end)
+
+	Trigger.OnKilledOrCaptured(USSRFC, function()
+		if not player.IsObjectiveCompleted(UseSpyObjective) then
+			player.MarkFailedObjective(UseSpyObjective)
+		end
+	end)
+end
+
+
+Tick = function()
+	ussr.Cash = 5000
+end
+
+TriggerRevealUSSRBase = function()
+	Trigger.OnEnteredProximityTrigger(LowerBaseWP.CenterPosition, WDist.FromCells(10), function(a, id)
+		if a.Owner == player then
+			Trigger.RemoveProximityTrigger(id)
+			local cam = Actor.Create("Camera", true, { Owner = player, Location = revealLowerBase.Location })
+			Trigger.AfterDelay(DateTime.Seconds(15), cam.Destroy)
+		end
+	end)
+	Trigger.OnEnteredProximityTrigger(UpperBaseWP.CenterPosition, WDist.FromCells(10), function(a, id)
+		if a.Owner == player then
+			Trigger.RemoveProximityTrigger(id)
+			local cam = Actor.Create("Camera", true, { Owner = player, Location = revealUpperBase.Location })
+			Trigger.AfterDelay(DateTime.Seconds(15), cam.Destroy)
+		end
+	end)
+end	
+
+TriggerExtractKosygin = function()
+	Trigger.OnEnteredProximityTrigger(KosyginExtractPoint.CenterPosition, WDist.FromCells(10), function(actor, triggerflee)
+		if actor.Type == kosyginType then
+			heli = Reinforcements.ReinforceWithTransport(player, ExtractionHelicopterType, nil, ExtractionPath)
+			Trigger.RemoveProximityTrigger(triggerflee)
+			Trigger.AfterDelay(DateTime.Seconds(10), function() 
+				player.MarkCompletedObjective(KosyginSurviveObjective)
+				player.MarkCompletedObjective(ExtractObjective)
+				Media.PlaySpeechNotification(player, "ObjectiveMet")
+			end)
+		end
+	end)
+end
+
+
+
+WorldLoaded = function()
+	player = Player.GetPlayer("Greece")
+	ussr = Player.GetPlayer("USSR")
+
+	Camera.Position = DefaultCameraPosition.CenterPosition
+	
+	UseSpyObjective = player.AddPrimaryObjective("Infiltrate the Soviet command center and contact Kosygin")
+	KosyginSurviveObjective = player.AddPrimaryObjective("Kosygin must survive")
+
+	Trigger.OnPlayerLost(player, MissionFailed)
+	Trigger.OnPlayerWon(player, MissionAccomplished)
+
+	InitialAlliedReinforcements()
+	InfiltrateForwardCenter()
+	TriggerRevealUSSRBase()
+	TriggerExtractKosygin()
+	ActivateAI()
+
```

Comment:
Remove space

Reply:
Thank you for your review. Unused space removed. 
[4b4c38e](https://github.com/OpenRA/OpenRA/pull/18695/commits/4b4c38e8bed864fd0e1a1d749bc44fef1960b7b9)